iidm/iidm-api/src/main/java/com/powsybl/iidm/network/RatioTapChanger.java

```diff
@@ -7,25 +7,69 @@
 package com.powsybl.iidm.network;
 
 /**
- * A ratio tap changer that is associated to a transformer to control the voltage.
+ * A ratio tap changer that is associated to a transformer to control the ratio.
```

Comment:
The ratio?

Reply:
This formulation to be consistent with PhaseTapChanger javadoc. 
Change to "... to control the voltage or reactive power"?

Reply:
I think there is an issue in phase tap changer javadoc.